Content.Shared/Interaction/Components/UnremoveableComponent.cs

```diff
@@ -1,8 +1,13 @@
+using Content.Shared.Whitelist;
 using Robust.Shared.GameStates;
 
 namespace Content.Shared.Interaction.Components
 {
     [RegisterComponent]
     [NetworkedComponent]
-    public sealed class UnremoveableComponent : Component {}
+    public sealed class UnremoveableComponent : Component 
+	{
+        [DataField("deleteOnDrop")]
+        public bool DeleteOnDrop = true;
```

Comment:
Add a comment on this and clarify it's only for hand drops not container removals please.

Reply:
added a comment, items inside can still be removed without being deleted. If target is gibbed however with items inside their backpack and backpack has DeleteOnDrop true then items inside are lost, if backpack has DeleteOnDrop = false then the backpack loses its unremoveable component.